Dev

divyaremadevi opened the pull request:
Hi Nishin,

We have completed the following functionalities. 
1. Login
2. Logout
3. Listing of customers, vacation request and daily status.

Please do verify and let us know your feedback.

Thanks
Divya, Remya, Alley

nishintn commented on controllers/DailyStatusController.js:
```diff
@@ -1,2 +1,64 @@
+
+    //********************************To populate Hour Dropdown*****************************//
+    $scope.hourList = hourList();
+    function hourList() {
```
Better we move this to a Util Service.

nishintn commented on controllers/DailyStatusController.js:
```diff
@@ -1,2 +1,64 @@
+    //******************************To populate Date Dropdown*******************************//
+    $scope.dateList = getDateList();
+
+    function getDateList() {
```
This can be a reusable function and hence suggest to move to Util service.

nishintn commented on controllers/Customer.js:
```diff
@@ -1,2 +1,23 @@
+    //********************************To populate customer details****************************//
+    BindCustomer();
+    function BindCustomer() {
+        $http.get("./shared/json/Customer.JSON")
```
Suggest to have a load JSON service and load it there.

nishintn commented on controllers/DailyStatusController.js:
```diff
@@ -1,2 +1,64 @@
+    }
+
+    //************************To populate ActivityType Dropdown******************************//
+    $http.get('./shared/json/ActivityType.JSON')
```
Suggest to have a load JSON service and load it there.

nishintn commented on controllers/DailyStatusController.js:
```diff
@@ -1,2 +1,64 @@
+    };
+
+    //******************************To fill Daily Status List******************************//
+    $http.get('./shared/json/DailyStatus.JSON')
```
Suggest to have a load JSON service and load it there.

nishintn commented on controllers/DashboardController.js:
```diff
@@ -0,0 +1,14 @@
+    if (userSession.isLogged) {
+        
+        $rootScope.userName = userSession.username;
+        $rootScope.showUser = { 'visibility': 'visible' };
```
Why dont we set a boolean value rather than a text?

nishintn commented on controllers/LoginController.js:
```diff
@@ -0,0 +1,42 @@
+
+    
+    if (!User.isLogged) {
+        $rootScope.showUser = { 'visibility': 'hidden' };
```
Why dont we set a boolean value rather than a text?

nishintn commented:
@divyaremadevi ,
Added some comments. Else looks good.

divyaremadevi commented:
Hi Nishin,

We have changed the code, based on code review and published the application. Find out the application in the link http://alleysurendran.github.io/AngularAPP. 

Implemented functionalities:
      1. Login
      2. Logout
      3. Customer Listing and filtering
      4. Daily Status listing, filtering and saving for admin and normal user
      5. Vacation listing, saving and year based filtering

Login credentials:
      1. Username: alley@qburst.com  (ADMIN)
          Password:  qburst
      2. Username: remya@qburst.com
          Password:  qburst@123
      3. Username: divya@qburst.com
          Password:  qbtech

Please do verify the code and let us know your feedback.

Thanks
Divya, Remya, Alley
